src/applications/disability-benefits/all-claims/content/evidenceTypesBDD.jsx

```diff
@@ -10,24 +26,50 @@ export const evidenceTypeTitle = (
 );
 
 export const privateMedicalRecords = 'Private medical records';
-export const evidenceLayStatements = `Additional VA forms, medical records,
-  separation documents (DD Form 214), supporting (lay) statements, or other
-  evidence`;
+
+export const bddShaOtherEvidence = (
+  <>
+    <strong>
+      Required Separation Health Assessment - Part A Self-Assessment
+    </strong>{' '}
+    or other documents like your DD214, supporting (lay) statements, or other
```

Comment:
I don't think an active duty service member (applying for a BDD) would have a DD214 (discharge papers)?

Reply:
Again, this content was reviewed by OCTO so we feel confident we can leave it as-is.